.claude/scripts/setup.sh

```
@@ -0,0 +1,200 @@
+        fi
+    else
+        echo "✓ POLARION_TOKEN configured"
+
+        # Ping Polarion in verify mode
+        if [[ "$VERIFY_MODE" == true ]]; then
+            echo
+            echo "Testing Polarion connection..."
+
+            # Load .env
+            set -a
+            source "$REPO_ROOT/.env"
```

Shellcheck (0.11.0)

[info] Not following: ./.env was not specified as input (see shellcheck -x).

(SC1091)